defi05/javascript/lazor.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Automata</title>
+    <meta charset="utf-8">
+</head>
+<body>
+    <canvas></canvas>
+    <script type="text/javascript">
+const v=(  // canVas
+ document.children[0].children[1].children[0]);
```

Comment:
Ça marche pas ici (`v.getContext is not a function`). Avec `document.querySelector('canvas')` ça marche, mais ça desobfusque un peu malheureusement